README.md

```diff
@@ -1 +1,69 @@
 # Goi
+
+## Deck Creation DSL
+### Example
+```
+Create Deck French with Tags language, hard, midterm using Style mystyle:  
+(1) Front:Back
+(2) Front:Back
+(3) Front:Back
+
+Create Style mystyle:
+Color = Red
+Direction = Horizontal
+Align = Center
+```
+
+### EBNF
+```
+CREATE_DECK ::= “Create Deck” NAME (“with Tags“ TAGS | “using Style” STYLE)*”:” DECK
```

Comment:
I think this is more clear that we want tags and styles to be optional:
CREATE_DECK ::= “Create Deck” NAME (“with Tags“ TAGS)? (“using Style” STYLE*)? :” DECK

Reply:
added spaces after “with Tags “ and “using Style ”:
CREATE_DECK ::= “Create Deck” NAME (“with Tags “ TAGS)? (“using Style ” STYLE*)? :” DECK

Reply:
Mhmm I see what you're getting at, but with the syntax you suggest, a user must define tags before styles, which I find pretty restrictive an unintuitive for the user. For example: "Create Deck French using Style style1 with Tags hard" would be wrong in your syntax.

Let me know what you think!

Reply:
I see. But we don't want "with Tags " and "using Style " to be repeated and "*" indicates that you can repeat them many times?

Reply:
Ah true, I'll update that